source/frontend/src/features/mapSideBar/consolidation/AddConsolidationView.tsx

```diff
@@ -0,0 +1,219 @@
+import { FieldArray, Formik, FormikHelpers, FormikProps } from 'formik';
+import { Tab } from 'react-bootstrap';
+import { FaInfoCircle } from 'react-icons/fa';
+import { toast } from 'react-toastify';
+import styled from 'styled-components';
+import * as Yup from 'yup';
+
+import { ReactComponent as ConsolidateSubdivideIcon } from '@/assets/images/subdivisionconsolidation.svg';
+import { Form } from '@/components/common/form';
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { H2 } from '@/components/common/styles';
+import { IMapSelectorContainerProps } from '@/components/propertySelector/MapSelectorContainer';
+import { StyledTabView } from '@/components/propertySelector/PropertySelectorTabsView';
+import { PropertySelectorPidSearchContainerProps } from '@/components/propertySelector/search/PropertySelectorPidSearchContainer';
+import PropertySearchSelectorPidFormView from '@/components/propertySelector/search/PropertySelectorPidSearchView';
+import { ApiGen_Concepts_Property } from '@/models/api/generated/ApiGen_Concepts_Property';
+import { exists } from '@/utils';
+
+import MapSideBarLayout from '../layout/MapSideBarLayout';
+import { AddressForm, PropertyForm } from '../shared/models';
+import SelectedOperationProperty from '../shared/operations/SelectedOperationProperty';
+import SelectedOperationPropertyHeader from '../shared/operations/SelectedOperationPropertyHeader';
+import SidebarFooter from '../shared/SidebarFooter';
+import AddConsolidationMarkerSynchronizer from './AddConsolidationMarkerSynchronizer';
+import { ConsolidationFormModel } from './AddConsolidationModel';
+
+export const AddConsolidationYupSchema = Yup.object().shape({
```

Comment:
isn't the regular pattern to have the yup schema on it's own file? this is fine too just trying to be consistent.

Reply:
I think it would depend on if it can be reused as well as general readability. In this case, I think is small enough to not affect the second criteria.